### ISSUE TITLE
resizeScaling - remove from POM

### ISSUE THREAD

**Issue by FlorianLudwig** [GreyRook/POM#6]
resizeScaling is currently part of the code drop.

But does it - architecturally speaking - belong into POM?

**Comment by FabianElsmer** [comment]
No, its GOWN and only relevant for our inheriting objects, we should simplify this function and simply override it in the prototype with the resizeScaling one, wherever we want to use it

**Comment by FlorianLudwig** [comment]
I agree, lets drop it
